### PR TITLE
[binding] support streaming decoding

### PR DESCRIPTION
**Pull request by robin1001**

### CODE REVIEW

**Review by pengzhendong — APPROVED** [review]

File: runtime/core/frontend/feature_pipeline.h [hunk 1]
```diff
@@ -91,6 +92,10 @@ class FeaturePipeline {
     return input_finished_ && (frame == num_frames_ - 1);
   }
 
+  int NumQuquedFrames() const {
```
Comment (xingchensong) [inline runtime/core/frontend/feature_pipeline.h:95]: NumQueuedFrames

File: runtime/core/decoder/asr_decoder.cc [hunk 2]
```diff
@@ -73,12 +75,17 @@ void AsrDecoder::Rescoring() {
...
-DecodeState AsrDecoder::AdvanceDecoding() {
+DecodeState AsrDecoder::AdvanceDecoding(bool block) {
   DecodeState state = DecodeState::kEndBatch;
   model_->set_chunk_size(opts_.chunk_size);
   model_->set_num_left_chunks(opts_.num_left_chunks);
   int num_requried_frames = model_->num_frames_for_chunk(start_);
   std::vector<std::vector<float>> chunk_feats;
+  // Return immediately if we do not want to block
+  if (!block && !feature_pipeline_->input_finished() &&
+      feature_pipeline_->NumQuquedFrames() < num_requried_frames) {
```
Comment (xingchensong) [inline runtime/core/decoder/asr_decoder.cc:86]: NumQueuedFrames

File: runtime/core/frontend/feature_pipeline.h (hunk 1 above)
Comment (robin1001) [reply runtime/core/frontend/feature_pipeline.h:95]: 火眼金睛

File: runtime/binding/python/README.md [hunk 3]
```diff
@@ -1,23 +1,33 @@
...
+This is a python binding of WeNet.
+
+WeNet is a production first and production ready end-to-end speech recognition toolkit.
+
+The best things of the binding are:
+
+1. Multiple languages supports, including English, Chinese. Other languages are in development.
+2. Non-streaming and streaming API
+3. N-best, contextual biasing, and timestamp supports, which are very important for speech productions.
+4. Alignment support. You can get get phone level alignments this tool.
```
[the remark this answers is not in the document]
Comment (robin1001) [reply runtime/binding/python/README.md:12]: okay

**Pull request merged**